api/kubernetes/cli/namespace.go

```diff
@@ -205,3 +257,37 @@ func (kcl *KubeClient) DeleteNamespace(namespace string) error {
 	}
 	return fmt.Errorf("namespace %s not found", namespace)
 }
+
+// CombineNamespacesWithResourceQuotas combines namespaces with resource quotas where matching is based on "portainer-rq-"+namespace.Name
+func (kcl *KubeClient) CombineNamespacesWithResourceQuotas(namespaces map[string]portaineree.K8sNamespaceInfo, w http.ResponseWriter) (map[string]portainer.K8sNamespaceInfo, *httperror.HandlerError) {
+	resourceQuotas, err := kcl.GetResourceQuotas("")
+	if err != nil {
+		return nil, httperror.InternalServerError("an error occurred during the CombineNamespacesWithResourceQuotas operation, unable to retrieve resource quotas from the Kubernetes for an admin user. Error: ", err)
+	}
+
+	if len(*resourceQuotas) > 0 {
+		return kcl.UpdateNamespacesWithResourceQuotas(namespaces, *resourceQuotas), nil
+	}
+
+	return namespaces, nil
+}
+
+// CombineNamespaceWithResourceQuota combines a namespace with a resource quota prefixed with "portainer-rq-"+namespace.Name
+func (kcl *KubeClient) CombineNamespaceWithResourceQuota(namespace portaineree.K8sNamespaceInfo, w http.ResponseWriter) *httperror.HandlerError {
+	resourceQuota, err := kcl.GetPortainerResourceQuota(namespace.Name)
+	if err != nil && !k8serrors.IsNotFound(err) {
+		return httperror.InternalServerError(fmt.Sprintf("an error occurred during the CombineNamespaceWithResourceQuota operation, unable to retrieve the resource quota associated with the namespace: %s for a non-admin user. Error: ", namespace.Name), err)
+	}
+	namespace.ResourceQuota = resourceQuota
+
+	return response.JSON(w, namespace)
+}
+
+func (kcl *KubeClient) BuildNonAdminNamespacesMap() map[string]struct{} {
```

Comment:
```suggestion
func (kcl *KubeClient) buildNonAdminNamespacesMap() map[string]struct{} {
```

This is likely an internal function for usage within the cli only?